{"name": "test_get_invalid_token", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [WinError 2] El sistema no puede encontrar el archivo especificado: '../.env' -> '../.env.bak'", "trace": "@pytest.mark.negative\n    def test_get_invalid_token():\n>       Login().login(USER, PASSWORD)\n\ntests\\test_get_list_comments.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nhelpers\\login.py:17: in login\n    for token in fileinput.FileInput(\"../.env\", inplace=1):\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\fileinput.py:249: in __next__\n    line = self._readline()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <fileinput.FileInput object at 0x000001813A370F10>\n\n    def _readline(self):\n        if not self._files:\n            if 'b' in self._mode:\n                return b''\n            else:\n                return ''\n        self._filename = self._files[0]\n        self._files = self._files[1:]\n        self._startlineno = self.lineno()\n        self._filelineno = 0\n        self._file = None\n        self._isstdin = False\n        self._backupfilename = 0\n        if self._filename == '-':\n            self._filename = '<stdin>'\n            if 'b' in self._mode:\n                self._file = getattr(sys.stdin, 'buffer', sys.stdin)\n            else:\n                self._file = sys.stdin\n            self._isstdin = True\n        else:\n            if self._inplace:\n                self._backupfilename = (\n                    os.fspath(self._filename) + (self._backup or \".bak\"))\n                try:\n                    os.unlink(self._backupfilename)\n                except OSError:\n                    pass\n                # The next few lines may raise OSError\n>               os.rename(self._filename, self._backupfilename)\nE               FileNotFoundError: [WinError 2] El sistema no puede encontrar el archivo especificado: '../.env' -> '../.env.bak'\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\fileinput.py:343: FileNotFoundError"}, "start": 1659564457953, "stop": 1659564459339, "uuid": "b81ead60-9f21-4ff1-9a0b-31752f2368db", "historyId": "cc3fbb98e0de1e99b951fe7681a1cd60", "testCaseId": "247ad6aab1f0ce4b327af4dbe5adbe73", "fullName": "tests.test_get_list_comments#test_get_invalid_token", "labels": [{"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_list_comments"}, {"name": "host", "value": "LAPTOP-PKT7OBP8"}, {"name": "thread", "value": "15816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_list_comments"}]}